.github/workflows/submitty_ci.yml

```diff
@@ -273,333 +274,197 @@ jobs:
           run: sudo apt-get install -y shellcheck
         - name: Run ShellCheck
           run: python3 run_shellcheck.py  # Uses the default Python installed with Ubuntu
+    e2eSelenium:
```

Comment:
It might be good to add some whitespace here to make it clear that this is a new job